core/src/main/java/tc/oc/pgm/spawns/SpawnMatchModule.java

```diff
@@ -269,8 +268,15 @@ public void matchEnd(final MatchFinishEvent event) {
     }
   }
 
+  //  // TODO: this should only be for initial spawn
```

Comment:
Actually this can be safely removed, since the PlayerSpawnLocationEvent does the exact same thing and is called in NMS right after the SportPaper PlayerInitialSpawnEvent.